CPS-0003/README.md

```diff
@@ -0,0 +1,141 @@
+---
+CPS: 3
+Title: Smart Tokens
+Status: Open
+Category: Tokens
+Authors:
+    - Istvan Szentandrasi <szist@zoeldev.com>
+Proposed Solutions: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/382
```

Comment:
There are certainly several other "conversations" around smart tokens at this point as I believe we have no less than 3 numbered CIPs that seek to address this behavior. Should they be included here as part of the discussion or should they be referenced somewhere in the body of the document itself?

Reply:
@Crypto2099 since the CPS at some point will include all those as CIP "mapping" links, I think the body content should enumerate the 3 relevant CIPs which in turn will have their own PR discussion links... and will have others added as time goes on, perhaps.  If & when each of those CIP PRs is merged, they can be replaced by CIP document links as we've done in other cases.  Once we can target a section to add these (at the meeting in the next hour) I'll update the content as such.

Reply:
Addressed with https://github.com/cardano-foundation/CIPs/pull/947/commits/565e374d65805381cce2bff27eaae1b5e2a989a1